docs/framework/docker/aspnetmvc.md

```diff
@@ -0,0 +1,241 @@
+
+```
+REPOSITORY                    TAG                 IMAGE ID            CREATED             SIZE
+mvcrandomanswers              latest              86838648aab6        2 minutes ago       8.104 GB
+```
+
+The IMAGE ID will be different on your machine. Now, let's run the applicaton.
+
+## Start a container
+
+You start a container by executing the follows `docker run` command:
+
+```
+docker run -d -p 8000:8000 --name randomanswers mvcrandomanswers
+```
+
+The `-d` argument tells Docker to start the image in detached mode. That
+means the Docker image runs disconnected from the current shell.
+
+The `-p 8000:8000` argument tells Docker how to map incoming ports. In this
+example, we're using port 8000 on both the host and the container.
+
+The `--name randomanswers` gives a name to the running container. You can use
+this name instead of the container ID in most commands.
+
+The `mvcrandomanswers` is the name of the image to start.
+
+## Verify in the browser
+
+> [!NOTE]
+> With the current release, you can't use `http://localhost` to browse
+> to your site. This is because of a known behavior in WinNAT, and it will
+> be resolved in the future. Until that is addressed, you need to use
+> the IP address of the container.
+
+Once the container starts, you'll need to find its IP address so that you
+can connect to your running container from a browser. You use the `docker exec`
+command to do that:
+
+```
+docker exec randomanswers ipconfig
+```
+
+You will see an output similar to this:
+
+```
+Windows IP Configuration
+
+
+Ethernet adapter Ethernet 2:
+
+   Connection-specific DNS Suffix  . :
+   Link-local IPv6 Address . . . . . : fe80::91d:ce97:dd27:460d%5
+   IPv4 Address. . . . . . . . . . . : 172.31.194.61
+   Subnet Mask . . . . . . . . . . . : 255.240.0.0
+   Default Gateway . . . . . . . . . : 172.16.0.1
+```
+
+You can connect to the running container using the IPv4 
+address and the configured port (8000), `http://172.31.194.61:8000`
+in the example shown. Type that URL into your browser, and you should
+see the running site.
+
+> [!NOTE]
+> Some VPN or proxy software may prevent you from navigating to your site.
+> You can temporarily disable it to make sure your container is working.
+
+The sample directory on GitHub contains a 
+[PowerShell script](https://github.com/dotnet/core-docs/tree/master/samples/framework/docker/MVCRandomAnswerGenerator/run.ps1)
+that executes these commands for you. Open a PowerShell window, change directory to
+your solution directory, and type:
+
+```
+./run.ps1
+```
+
+It builds the image, displays the list of images on your machine, starts
+a container, and displays the IP address for that container. 
+
+When you are done, and you want to stop your container, issue a `docker
+stop` command:
+
+```
+docker stop randomanswers
+```
+
+To remove the container, issue a docker rm command:
+
+```
+docker rm randomanswers
+```
+
+## Summary
+
+In this topic, you've seen the steps to move an existing
```

Comment:
> In this topic, you've seen the steps to move an existing ASP.NET MVC application to run in a Windows container. 

In this topic, you've seen the steps to move and run an existing ASP.NET MVC application in a Windows Server Container.

> Running an existing application does not require any changes to your application.

> Your existing ASP.NET MVC applications are container-ready.

While it didn't this time around.. I am not sure we have high confidence that this is a true statement for all applications. I would feel more comfortable either taking this out or softening what I somewhat equate to as a 'promise'.